### ISSUE TITLE
Book 1 (v1.1.0) is missing section on loops

### ISSUE THREAD

**Issue by surugu** [realpython/support#305]
Book 1 v1.0.2 has an assignment called "Convert temperatures" on page 55, followed by the section "Run in circles" on page 56. This is the section that introduces loops.

Book 1 v1.1.0 omits both the assignment and the section and instead goes straight to "Assignment: Track your investments", the first assignment to use loops.

**Comment by mjhea0** [comment]
I'm seeing otherwise...

![screen shot 2016-08-24 at 8 21 52 am](https://cloud.githubusercontent.com/assets/2018167/17934050/b5069cc8-69d3-11e6-9bd6-f7b188e1e076.png)

**Comment by surugu** [comment]
This is what it looks like in my copy of 1.1.0 ![image](https://cloud.githubusercontent.com/assets/18073069/17952100/21d9fe4a-6a2d-11e6-90bf-7b0d4601807e.png)

I did something I didn't do last time, and downloaded the zip again and compared them. I didn't think to do so because the version number didn't change and the changelog didn't mention it. 
![image](https://cloud.githubusercontent.com/assets/18073069/17952304/d6a9d704-6a2e-11e6-9f65-353aba605f1e.png)

Going by the timestamps a fixed version was uploaded four days later. So, it is fixed now, but the version I downloaded when I first got the email was affected. I guess this can be closed, with the knowledge that anyone with a wonky copy should just redownload.
